### ISSUE TITLE
Should we split up en.json into separate files?

### ISSUE THREAD

**Issue by dunyakirkali** [vadymmarkov/Fakery#118]
The current structure (a single `en.json` file) makes it difficult to contribute to this repository.

When there is more than 1 PR pending, once the first one gets merged it always creates merge-conflicts for the one after

**Comment by vadymmarkov** [comment]
That's true, but I'm afraid it will make the parsing logic more complicated. Also, if we do it for `en.json` file it means we should probably do it for other locales, which could also be a problem.

**Comment by dunyakirkali** [comment]
True that!
I’ll try to create something as a POC and discuss with u further how we can tackle the issue
